tests/virt_autotest/virsh_external_snapshot.pm

```diff
@@ -51,6 +51,10 @@ sub run_test {
     my @vm_hostnames_inactive_array = split(/\n+/, $vm_hostnames_inactive);
 
     foreach my $guest (keys %xen::guests) {
+        #RAW do not support Snapshot test
+        my $guest_disk_type = script_output("virsh dumpxml $guest | grep 'driver name' | sed \"s/'/ /g\"| awk \'{ print \$5 }\'");
+        return if ($guest_disk_type =~ /raw/);
```

Comment:
Just return without any hint, such as printing some warning?

Reply:
For now, there just only be the qcow2(sles12+) and raw(sles11sp4) as guest dist type.

Reply:
I mean this kind of message like "recordinfo(skip snapshot tests on sles11sp4 ...)", then the user may notice that we does not run the snapshot test in the test case.

Reply:
Great! good idea,  I will update code as your mentioned.